superset/assets/src/views/chartList/ChartList.tsx

```diff
@@ -0,0 +1,316 @@
+        },
+      }: any) => <a href={datasource_link}>{datasource_name_text}</a>,
+      Header: t('Datasource'),
+      accessor: 'datasource_name_text',
+      sortable: true,
+    },
+    {
+      Cell: ({
+        row: {
+          original: { changed_by_name, changed_by_url },
+        },
+      }: any) => <a href={changed_by_url}>{changed_by_name}</a>,
+      Header: t('Creator'),
+      accessor: 'creator',
+      sortable: true,
+    },
+    {
+      Cell: ({
+        row: {
+          original: { changed_on },
+        },
+      }: any) => (
+          <span className='no-wrap'>{moment(changed_on).fromNow()}</span>
+        ),
+      Header: t('Last Modified'),
+      accessor: 'changed_on',
+      sortable: true,
+    },
+    {
+      Cell: ({ row: { state, original } }: any) => {
+        const handleDelete = () => this.handleChartDelete(original);
+        const handleEdit = () => this.handleChartEdit(original);
+        if (!this.canEdit && !this.canDelete) {
+          return null;
+        }
+
+        return (
+          <span className={`actions ${state && state.hover ? '' : 'invisible'}`}>
+            {this.canDelete && (
+              <ConfirmStatusChange
+                title={t('Please Confirm')}
+                description={<>{t('Are you sure you want to delete')} <b>{original.slice_name}</b>?</>}
+                onConfirm={handleDelete}
+              >
+                {(confirmDelete) => (
+                  <span
+                    role='button'
+                    className='action-button'
+                    onClick={confirmDelete}
+                  >
+                    <i className='fa fa-trash' />
+                  </span>
+                )}
+              </ConfirmStatusChange>
+            )}
+            {this.canEdit && (
+              <span
+                role='button'
+                className='action-button'
+                onClick={handleEdit}
+              >
+                <i className='fa fa-pencil' />
+              </span>
+            )}
+          </span>
+        );
+      },
+      Header: 'Actions',
+      id: 'actions',
+    },
+  ];
+
+  public hasPerm = (perm: string) => {
+    if (!this.state.permissions.length) {
+      return false;
+    }
+
+    return Boolean(this.state.permissions.find((p) => p === perm));
+  }
+
+  public handleChartEdit = ({ id }: { id: number }) => {
+    window.location.assign(`/chart/edit/${id}`);
+  }
+
+  public handleChartDelete = ({ id, slice_name }: Chart) => {
+    SupersetClient.delete({
+      endpoint: `/api/v1/chart/${id}`,
+    }).then(
+      (resp) => {
+        const { lastFetchDataConfig } = this.state;
+        if (lastFetchDataConfig) {
+          this.fetchData(lastFetchDataConfig);
+        }
+        this.props.addSuccessToast(t('Deleted') + ` ${slice_name}`);
+      },
+      (err: any) => {
+        this.props.addDangerToast(t('There was an issue deleting') + `${slice_name}`);
```

Comment:
looks like this is missing a space before slice_name. Although I don't think this translation will actually work anyway. i believe there's a way to put `%s` or something in the translation string so that the full sentence gets parsed for translation. You could look in the code to find it

Reply:
Looks like there's docs around this: https://github.com/apache-superset/superset-ui/tree/master/packages/superset-ui-translation

Seems simple enough